### ISSUE TITLE
feature: add an install command

### ISSUE THREAD

**Issue by doctordesh** [kardianos/govendor#89]
I might have misunderstood some parts of the readme, it so, please excuse me. Possible duplicate: https://github.com/kardianos/govendor/issues/35

I would like to have a command that installs the dependencies specified in the `vendor.json`, at the version specified by `revision`.

At the moment, there is just `add` and `update` but if I start up my project on a new computer, I don't want to update my dependencies, right?

Install would read the `vendor.json` file and install the dependencies specified. If this works I should be able to delete everything in my `vendor` directory (except `vendor.json`), run `govendor install` and it should be back again, the exact same code.

**Comment by kardianos** [comment]
@doctordesh This is what I have planned:
https://github.com/kardianos/govendor/blob/master/DESIGN.md

Feedback welcome.

**Comment by kardianos** [comment]
This is implemented in the fetch branch. Once that feature branch is complete it will be merged into master.

**Comment by doctordesh** [comment]
Awesome! Thanks :)
